views/js/qtiCreator/widgets/interactions/sliderInteraction/Helper.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020  (original work) Open Assessment Technologies SA;
+ */
+
+define([
+    'i18n',
+    'ui/feedback',
+    'ui/dialog/confirm',
+], function (__, feedback, dialogConfirm) {
+
+    return {
+        /**
+         * Validate all parameters and check that slider has correct values
+         * @param interaction
+         * @param response
+         * @param currentResponse
+         */
+        responseManager: function responseManager(interaction, response, currentResponse) {
+            const lowerBound = interaction.attributes.lowerBound;
+            const upperBound = interaction.attributes.upperBound;
+
+            let validResponse = 0;
+            let errMessage = '';
+            let warnQuestion = '';
+
+            // if response is bigger than allowed
+            if (lowerBound > upperBound) {
```

Comment:
Maybe its more cleaner to use a Switch

```
switch (true) {
  case (val < 1000): /* do something */ break;
  case (val < 2000): /* do something */ break;
  ...
  case (val < 30000): /* do something */ break;
}
```

More examples: https://stackoverflow.com/questions/6665997/switch-statement-for-greater-than-less-than

Reply:
1. `if` faster
2. I thought about the `steps`, what if we need more conditions here? then switch not so clear